src/app/shared/detail/detail-container/detail-container.component.html

```diff
@@ -1 +1,10 @@
-<p>detail-container works!</p>
+<mu-header></mu-header>
+<article>
+  <section class="article-container">
+    <img class="article-profile-image" src="{{heroData.thumbnail.path}}.{{heroData.thumbnail.extension}}"
+         alt="profile_image">
+    <div *ngIf='heroData.description !=""; else no_desc' class = "hero-desc">{{heroData.description}}</div>
+  </section>
+</article>
```

Comment:
also there was a separate detail component right? and you were supposed to pass data to that. this is supposed to be the container